### ISSUE TITLE
ADDON BAR Disappears

### ISSUE THREAD

**Issue by aracopelli** [Geek-in-Training/Firefox-Addon-Bar--Restored#11]

**Comment by aracopelli** [comment]
Whenever I restart Firefox 29.0.1 the addon bar is gone and I have to go into :customize" and reactivate it in the show/hide toolbars section.. Sometimes it just disappears even if Firefox has not been restarted if I open a new window. Apparently firefox is losing the settings for this toolbar. How can I avoid this problem? Truth is I love this addon and use it always...Help!
Thanks.

**Comment by Geek-in-Training** [comment]
I see you closed the issue.  I am curious how you solved it; did it just stop happening?

**Comment by aracopelli** [comment]
Thanks for replying.
Somehow this issue got 'closed" in error.The problem still exists on Firefox 29.0.1.
Whenever I restart Firefox 29.0.1 the addon bar is gone and I have to go into "customize" and reactivate it in the show/hide toolbars section.. Sometimes it just disappears even if Firefox has not been restarted if I open a new window. Apparently firefox is losing the settings for this toolbar. I have not been able to locate any settings for this extension in 'about config' that might explain this behavior. How can I avoid this problem? Truth is I love this addon and use it always...Help!
aracopelli.
Thanks.

p.s.  I also installed this extension on Firefox Aurora (separate profile) where it works perfectly.

**Comment by Geek-in-Training** [comment]
Are you using an older version of _The Addon Bar (Restored)_ than 3.2?

**Comment by aracopelli** [comment]
No...Using 3.2. There may be some interaction with another addon or theme that I haven't found yet. I will keep looking at it. Let me know if you come up with anything,, I will do likewise. 
Thanks

**Comment by Geek-in-Training** [comment]
Does it still occur if you make sure the addon bar is visible in all windows, and then restart Firefox?

**Comment by Geek-in-Training** [comment]
Note: <kbd>Ctrl</kbd> + <kbd>/</kbd> is the shortcut for quickly hiding/showing the addon bar. (<kbd>Cmd</kbd> + <kbd>/</kbd> on a Mac).

**Comment by aracopelli** [comment]
No, it is gone again on restart. This only happens in Firefox 29.0.1.. Where are the settings in about:config. I can check those.Thanks.--- notifications@github.com wrote:From: Geek in Training notifications@github.comTo: Geek-in-Training/Firefox-Addon-Bar--Restored Firefox-Addon-Bar--Restored@noreply.github.comCc: aracopelli forus@techemail.comSubject: Re: [Firefox-Addon-Bar--Restored] ADDON BAR Disappears (#11)Date: Mon, 26 May 2014 19:14:00 -0700Does it still occur if you make sure the addon bar is visible in all windows, and then restart Firefox?

**Comment by aracopelli** [comment]
I mean, Yes, it is gone again on restart. This only happens in Firefox 29.0.1.. Where are the settings in about:config. I can check those.Thanks.--- notifications@github.com wrote:From: Geek in Training notifications@github.comTo: Geek-in-Training/Firefox-Addon-Bar--Restored Firefox-Addon-Bar--Restored@noreply.github.comCc: aracopelli forus@techemail.comSubject: Re: [Firefox-Addon-Bar--Restored] ADDON BAR Disappears (#11)Date: Mon, 26 May 2014 19:14:00 -0700Does it still occur if you make sure the addon bar is visible in all windows, and then restart Firefox?

**Comment by aracopelli** [comment]
Had to switch extension to  "New Add-on Bar" by Leszek Życzkowski.
Thanks.

**Comment by Geek-in-Training** [comment]
Sorry for the late response, the relevant setting in about:config would be `extensions.GiTAddonBar.collapsed`.  I know you are using the other addon now, but could you send me a list of other addons you had installed? I would like to be able to figure out what was causing the problem. Thanks!

**Comment by aracopelli** [comment]
New computer and new installed Firefox. No longer have the extensions list that were installed on the old machine . If I experience this problem in the future I will let u know. During my investigation of the problem I narrowed it down to possibly a compatibility problem with another add-on called "Configuration Mania" but that's as far as I as I got.Thanks for your follow-up.Regards,aracopelli--- notifications@github.com wrote:From: Geek in Training notifications@github.comTo: Geek-in-Training/Firefox-Addon-Bar--Restored Firefox-Addon-Bar--Restored@noreply.github.comCc: aracopelli forus@techemail.comSubject: Re: [Firefox-Addon-Bar--Restored] ADDON BAR Disappears (#11)Date: Mon, 02 Jun 2014 23:10:46 -0700Sorry for the late response, the relevant setting in about:config would be extensions.GiTAddonBar.collapsed.  I know you are using the other addon now, but could you send me a list of other addons you had installed? I would like to be able to figure out what was causing the problem. Thanks!

**Comment by deepstblu** [comment]
Had this problem and adding the about:config entry fixed it for me. Thanks!

**Comment by sudofox** [comment]
I work in web hosting and use the Addon Bar for WorldIP. It disappeared, and I couldn't get it back, tried everything, even toggling the value in about:config and restarting Firefox and reinstalling the addon after removing it....Finally found the secret to making the bar come back: `Ctrl+/` ! Thank you so much. Should be a configurable option within the addon settings itself to hide or show it.

Addon Bar (Restored) 3.2.9 compat-fixed-4
Firefox 52.0.2 (64-bit) on Arch Linux
